package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.6",
   "description": "Passport strategy for Google OAuth 2.0",
   "main": "./lib/oauth2.js",
+  "typings": "lib/index",
```

Comment:
This is how we link a type definitions file (*.d.ts) so typescript can find it when importing the module into a project. You can optionally do the extension but it knows it's looking for a `*.d.ts` file so it's not required.

I named this index because that's the convention, but it could just as easily be named `oauth2.d.ts` if so desired.